contrib/dockerd-rootless-setuptool.sh

```diff
@@ -95,8 +95,8 @@ init() {
 			ERROR "- or simply log back in as the desired unprivileged user (ssh works for remote machines, machinectl shell works for local machines)"
 			exit 1
 		fi
-		export XDG_RUNTIME_DIR="/tmp/docker-$(id -u)"
-		mkdir -p "$XDG_RUNTIME_DIR"
+		export XDG_RUNTIME_DIR="$HOME/.docker/xrd"
```

Comment:
Was discussing naming with some colleagues; would `run` (as subdirectory) work?

```suggestion
		export XDG_RUNTIME_DIR="$HOME/.docker/run"
```

Reply:
done